### ISSUE TITLE
Humanise swap expiry moments (late nights, weekends)

### ISSUE THREAD

**Issue by tdg** [swapmyvote/swapmyvote#966]
Don't expire overnight etc

**Comment by baob** [comment]
Are you saying we shouldn't run the scheduled expiry overnight ?

**Comment by tdg** [comment]
Yes

**Comment by aspiers** [comment]
No.  If it runs first thing in the morning then effectively there's no significant difference to users, unless they were using the site during the night.  The point is for night hours not to count towards expiry at all, just like "you have 3 working business to respond" doesn't count the weekend.

**Comment by baob** [comment]
That's what I was thinking, it's less "humanise expiry moments" than "humanise swap expiry duration"

**Comment by baob** [comment]
And we could improve this by setting the duration on the the swap itself, so that swaps don't get prematurely expired by a change in the system-wide swap expiry hours. (Issue raised in #924)
